### ISSUE TITLE
Is this under MIT license?

### ISSUE THREAD

**Issue by darrylryan** [Differential/polymer-demo#12]
I can't find any license info in this repo, are we allowed to use it? Thanks.

**Comment by ryw** [comment]
Yes absolutely — I'll leave this open as a reminder to add the license.
